internal/app/dashboard/main.go

```diff
@@ -44,13 +45,13 @@ func dashboardMain() error {
 		return fmt.Errorf("error connecting to the machine service: %w", err)
 	}
 
-	// TODO(dashboard): enable the network config screen once it's ready (remove the screens override option)
-	d, err := dashboard.New(c, dashboard.WithAllowExitKeys(false),
-		dashboard.WithScreens(dashboard.ScreenSummary, dashboard.ScreenMonitor),
-	)
-	if err != nil {
-		return err
+	screens := []dashboard.Screen{dashboard.ScreenSummary, dashboard.ScreenMonitor}
+
+	// activate the network config screen only on metal platform
+	currentPlatform, _ := platform.CurrentPlatform() //nolint:errcheck
+	if currentPlatform != nil && currentPlatform.Name() == "metal" {
```

Comment:
maybe use the costant for the metal platform if that exists

Reply:
this place doesn't use the constant @utkuozdemir